server/routes/conversation.js

```diff
@@ -0,0 +1,16 @@
+const express = require("express");
+const router = express.Router();
+const protect = require("../middleware/auth");
+const {
+  createConversation,
+  getAllMessages,
+  sendMessage,
+  getAllConversations,
+} = require("../controllers/conversation");
+
+router.route("/create-conversation").post(protect, createConversation);
```

Comment:
```suggestion
router.route("/").post(protect, createConversation);
router.route("/").post(protect, sendMessage);
router.route("/:conversationId/messages").get(protect, getAllMessages);
router.route("/").get(protect, getAllConversations);
```
We can further simplify these routes

Reply:
two of the routes will now be the same URL, so I guess I will change send message to /send ?

Reply:
Good catch 😄